portal/v2/package.json

```
@@ -19,7 +19,7 @@
   },
   "scripts": {
     "start": "HTTPS=true react-app-rewired start",
-    "build": "react-app-rewired build",
+    "build": "react-app-rewired build && rm -rf ../../pkg/portal/assets/v2/build && mv -f build ../../pkg/portal/assets/v2/",
```

As I said above, `rm -rf` isn't a great option. Looks like we can directly override the build path as below.

```suggestion
    "build": "BUILD_PATH=../../pkg/portal/assets/v2/build react-app-rewired build",
```

---

Done